examples/mpi/lammps/Dockerfile

```diff
@@ -1,29 +1,41 @@
 # ch-test-scope: full
 FROM openmpi
-WORKDIR /
+WORKDIR /usr/local/src
 
 # Packages for building.
 RUN apt-get install -qy --no-install-recommends \
+    cmake \
     git \
     patch \
-    python-dev
+    python-dev \
```

Comment:
Can we use Python 3? (`python3-foo` and then `pip3` later)

Reply:
I'll change to python3 and retest.

Reply:
I will change to python3 and retest

Reply:
This required moving the install target to /usr/ as the resulting python3 lammps module wasn't being found. This is consistent with the direction we plan to go with the CentOS 8 base images .